### PR TITLE
tests: fix cc_oci_setup_shim test

### PR DESCRIPTION
**Pull request by jcvenegas**
This patch fixes a random issue for cc_oci-setup_shim unit
tests.

The issue was caused because a new function parameter
was added but the function definition was not updated
in the unit test. As a result the new parameter value
(int shim_flock_fd) got random values because was not properly
initialized and the test failed when the value was negative.

The function definition is updated according to the function
signature.

Signed-off-by: Jose Carlos Venegas Munoz <jose.carlos.venegas.munoz@intel.com>

### CODE REVIEW

File: tests/process_test.c [hunk 1]
```diff
@@ -194,32 +195,42 @@ START_TEST(test_cc_oci_setup_shim) {
...
-	ck_assert (! cc_oci_setup_shim (&config, tmpf1_fd, tmpf2_fd));
+	ck_assert (! cc_oci_setup_shim (&config, tmpf1_fd, tmpf2_fd, flock_fd));
 
 	g_free (config.console);
 	cc_oci_rm_rf (tmpf1);
 	cc_oci_rm_rf (tmpf2);
+	cc_oci_rm_rf (tmpf3);
+	if (tmpf1_fd > -1) { close (tmpf1_fd); }
+	if (tmpf2_fd > -1) { close (tmpf2_fd); }
+	if (flock_fd > -1) { close (flock_fd); }
```
Comment (devimc) [inline tests/process_test.c]: good catch , but I think that fds should be closed before remove them

**Comment by chavafg** [comment]
qa-failed

File: tests/process_test.c (hunk 1 above)
Comment (jcvenegas) [inline tests/process_test.c]: right!, fixed :)

**Comment by GabyCT** [comment]
qa-passed

**Comment by GabyCT** [comment]
There was a failure before mainly because of the nuc in our lab was shutdown

**Comment by gorozco1** [comment]
Thanks for the update @GabyCT

**Comment by gorozco1** [comment]
lgtm

**Comment by grahamwhaley** [comment]
The PR and commit messages don't really tell me what was fixed, or what the symptoms it fixes are (such as - what errors did we see). Can we try to add a bit more 'what and why' to the commits in future please :-)

**Comment by jcvenegas** [comment]
@grahamwhaley sure :) , description updated

**Comment by grahamwhaley** [comment]
Great, that helps a lot @jcvenegas 
lgtm

**Comment by sameo** [comment]
LGTM

**Pull request merged**